chapters/6-analysis.tex

```diff
@@ -147,13 +147,19 @@ \subsection{Fitting construction into the data workflow}
 in a unified script, separate from the analysis code, helps
 avoid this and ensures consistency across different outputs.
 
-\subsection{Integrating different data sources}
+\subsection{Combining different data sources}
```

Comment:
Why the change? data integration is the technical term, this is not a case of overly-difficult vocabulary

Reply:
I tested a re-structure where integration would come sooner, but it wasnt better, so I changed back but missed that I had committed this. I reverted back to original wording in 653b8453d228ecc48dc78ced1b068797676a3b7b